tutorials/parquet-catalog-demos/euclid-hats-parquet.md

```diff
@@ -0,0 +1,256 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.1
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# Euclid Quick Release 1: MER Catalogs in HATS Parquet
+
++++
+
+## Learning Goals
+By the end of this tutorial, you will:
+
+- Understand the format, partitioning, and schema of this dataset.
+- Be able to query this dataset for likely stars.
+
++++
+
+## Introduction
+
++++
+
+This notebook demonstrates accesses to a copy of the
+[Euclid Q1](https://irsa.ipac.caltech.edu/data/Euclid/docs/overview_q1.html) MER Catalogs
+that is in Apache Parquet format, partitioned according to the
+Hierarchical Adaptive Tiling Scheme (HATS), and stored in an AWS S3 bucket.
+Parquet is a file format that enables flexible and efficient data access by, among other things,
+supporting the application of both column and row filters when reading the data (very similar to a SQL query)
+so that only the desired data is loaded into memory.
+
+This is a single parquet dataset which comprises all three MER Catalogs
+-- MER, MER Morphology, and MER Cutouts -- which have been joined by Object ID.
+Their schemas (pre-join) can be seen at
+[Euclid Final Catalog description](http://st-dm.pages.euclid-sgs.uk/data-product-doc/dmq1/merdpd/dpcards/mer_finalcatalog.html).
+Minor modifications were made to the parquet schema to accommodate the join (de-duplicating column names)
+and for the HATS standard. These differences are shown below.
+
+HATS is a spatial partitioning scheme based on HEALPix that aims to
+produce partitions (files) of roughly equal size.
+This makes them more efficient to work with,
+especially for large-scale analyses and/or parallel processing.
+This notebook demonstrates the basics.
+
++++
+
+## Installs and imports
+
+```{code-cell}
+# !pip uninstall -y numpy pyerfa  # Helps resolve numpy>=2.0 dependency issues.
+# !pip install 'hats>=0.5' 'lsdb>=0.5' matplotlib numpy s3fs
+```
+
+```{code-cell}
+import os
+
+import dask.distributed
+import hats
+import lsdb
+import matplotlib.colors
+import matplotlib.pyplot as plt
+import numpy as np
+# NOTE: If you run into an error that starts with,
+# "A module that was compiled using NumPy 1.x cannot be run in NumPy 2.1.3 as it may crash.",
+# make sure you have restarted the kernel since doing `pip install`. Then re-run the cell.
+```
```

Comment:
You can move this into a narrative, e.g. tip section to warn users they may need to fix-up their envs.

````suggestion
```

```{tip}
If you run into an error that starts with,
"A module that was compiled using NumPy 1.x cannot be run in NumPy 2.1.3 as it may crash.",
make sure you have restarted the kernel since doing `pip install`. Then re-run the cell.
```
````